scripts/biomedical/VirtualMetabolicHuman/README.md

```diff
@@ -0,0 +1,136 @@
+# Importing VMH Metabolome and Microbiome data
+
+## Table of Contents
+
+- [Importing VMH Metabolome and Microbiome data](#importing-vmh-metabolome-and-microbiome-data)
+  - [Table of Contents](#table-of-contents)
+  - [About the Dataset](#about-the-dataset)
+  - [About the import](#about-the-import)
+
+## About the Dataset
+
+- ### Download URL
+
+The human metabolite and reaction data can be downloaded from the Virtual Metabolic Human database using their web [interface](https://www.vmh.life/#human/all).The gut microbiome data can be downloaded [here](https://www.vmh.life/#microbes/search). The data is in tab delimited format. The data is roughly in tab delimited format (see [Notes and Caveats](#notes-and-caveats) for additional information on formatting).
+
+- ### Overview
+
+This directory stores the scripts used to convert the datasets obtained from VHM into modified versions, for effective ingestion of data into the Data Commons knowledge graph.
+
+The VHM database explicity connects human metabolism with genetics and human-associated microbial metabolism. It consists of five interconnected resources: `Human metabolism`, `Gut microbiome`, `Disease`, `Nutrition` and `ReconMaps`. More information on the database can be found on their official [website](https://www.vmh.life/#home) and [manuscript](https://academic.oup.com/nar/article/47/D1/D614/5146204).
+For the knowledge graph, three files are imported:
+
+- <u>recon-store-metabolites-1.tsv</u> : contains information on chemical properties of metabolites, and identifiers for various databases to query the metabolites. In order to generate dcids, the keggID for each metabolite was converted to the ChEMBL ID, using the [bioservices packages](https://bioservices.readthedocs.io/en/master/compound_tutorial.html). For the metabolites, with missing keggIDs, the chemical formulae were used to generate the dcids.
```

Comment:
I thought we used iupacName not chemical formulae to generate dcid in cases for which there was no ChEMBLID

Reply:
I'm sorry about that, I fixed it. I'm using the iupac in the .py script but somehow wrote it incorrectly on the readme.